use-cases/node.js/run_jcl/scripts/zowe_run_jcl.js

```diff
@@ -157,15 +158,15 @@ const parms = {
  *   Give zos a chance to realize the dataset exists                *
  ********************************************************************/
 
-setTimeout(() => {return null;}, 2000);
+setTimeout(() => {}, 2000);
```

Comment:
The way `setTimeout` is being called may not have the intended effect. It will wait 2 seconds to execute `() => {}`, and the following code will execute immediately.

This comment also applies to the other usages of `setTimeout` in this file.

Reply:
Replaced with an actual delay.